app/assets/javascripts/expand.js

```diff
@@ -55,8 +57,9 @@ $(document).ready(function() {
 
       para.appendChild(update_button);
       para.appendChild(delete_button);
-      element.appendChild(title_header);
-      element.appendChild(para);
+      post_it.appendChild(title_header);
```

Comment:
Identifier 'post_it' is not in camel case.<br>Identifier 'title_header' is not in camel case.